test/basic-querying.test.js

```diff
@@ -153,6 +153,26 @@ describe('basic-querying', function() {
           done();
         });
     });
+    it('should query by ids and null condition', function(done) {
+      // due to some connectors not being assigning null value by default
+      // the query is filtered by the fields which may consist null value
+      // or not exist at all
+      User.findByIds([
+        createdUsers[0].id,
+        createdUsers[1].id],
+        {fields: {name: true, vip: true}}, function(err, users) {
```

Comment:
or i'm missing this: one of the user has an undefined property "vip" and the test should be named something like: `'should query by ids and condition, allowing results where conditions match null properties'`